tests/p2p/test_peer_id.py

```diff
@@ -144,26 +149,27 @@ def test_retry_connection(self):
         self.assertEqual(p.retry_interval, 5)
         self.assertEqual(p.retry_timestamp, 0)
 
-    def test_validate_certificate(self):
+    def test_validate_certificate(self) -> None:
         builder = TestBuilder()
         artifacts = builder.build()
-        protocol = artifacts.p2p_manager.server_factory.buildProtocol('127.0.0.1')
+        protocol = artifacts.p2p_manager.server_factory.buildProtocol(Mock())
```

Comment:
Why not `'127.0.0.1'`?

Reply:
`buildProtocol()` expects an `IAddress` object, not a string.

Reply:
Got it. Thanks!